### DISCUSSION TITLE
Ability to get shareable config presets from other organizations in Azure Devops

### DISCUSSION THREAD

**Discussion by ghost** [renovatebot/renovate#26146] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.83.2

### If you're self-hosting Renovate, select which platform you are using.

Azure DevOps (dev.azure.com)

### What is your question?

Consider a scenario where I am running renovate on multiple repositories across multiple organizations in Azure DevOps, but I would like to extend from multiple different shareable configurations in a separate organization. For example:

* Org: my-renovate-config  
    * Project: renovate-config  
        * Repository: renovate-config  
            * Folder: maven/default.json  
            * Folder: npm/default.json  
            * ...  

* Org: my-org-1
    * project: my-project-1
        * Repository: webgoat
            * inherits from: my-renovate-config/renovate-config/renovate-config//maven/
        * Repository: nodegoat
            * inherits from: my-renovate-config/renovate-config/renovate-config//npm/

Typically I feel like I could do this on Github (although I haven't tried yet), but I'd like to implement this solution on AZDO. The issue is the extra layer of hierarchy in AZDO. When providing the configuration:
```json
{
    "extends": [ "local>my-renovate-config/renovate-config/renovate-config//maven/" ]
}
```

It ends up erroring out due to the format provided as shown in the logs below.

### Logs (if relevant)

<details><summary>Logs</summary>

```

INFO: Repository has invalid config (repository=myorg/Test-Renovate)
       "error": {
         "validationError": "Cannot find preset's package (local>my-renovate-config/renovate-config/renovate-config)",
         "message": "config-validation",
         "stack": "Error: config-validation\n    at fetchPreset (/usr/local/lib/node_modules/renovate/lib/config/presets/index.ts:398:19)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at Object.resolveConfigPresets (/usr/local/lib/node_modules/renovate/lib/config/presets/index.ts:319:31)\n    at mergeRenovateConfig (/usr/local/lib/node_modules/renovate/lib/workers/repository/init/merge.ts:267:5)\n    at getRepoConfig (/usr/local/lib/node_modules/renovate/lib/workers/repository/init/config.ts:12:12)\n    at initRepo (/usr/local/lib/node_modules/renovate/lib/workers/repository/init/index.ts:52:12)\n    at Object.renovateRepository (/usr/local/lib/node_modules/renovate/lib/workers/repository/index.ts:55:14)\n    at attributes.repository (/usr/local/lib/node_modules/renovate/lib/workers/global/index.ts:184:11)\n    at start (/usr/local/lib/node_modules/renovate/lib/workers/global/index.ts:169:7)\n    at /usr/local/lib/node_modules/renovate/lib/renovate.ts:18:22"
       }


```

</details>

**Comment by rarkins** [comment]
I think the problem here is that in Azure Devops, the `endpoint` is configured to something like `https://dev.azure.com/YOUR-ORG/`. i.e. pinned to your org, unlikely in other platforms where the endpoint is e.g. `api.github.com`. Then this means that `local>` is local within your endpoint (org).

A solution to this could be for us to support an `azure>` prefix for presets, where the first part of following string will be the org name, allowing cross-org prefixes.
